tests/test_operations.cpp

```diff
@@ -2,35 +2,9 @@
 #include <catch2/catch.hpp>
 #include <iostream>
 
-#include "../app/TetWild.h"
-
 using namespace wmtk;
 
 TEST_CASE("edge_splitting", "[test_operation]")
 {
-    using namespace tetwild;
-
-    Parameters params;
-    params.l = 1 / 20.;
-    params.init();
-
-    Envelope envelope;
-    TetWild tetwild(params, envelope);
-
-    std::vector<VertexAttributes> vertices(4);
-    vertices[0].m_posf = Vector3f(0, 0, 0);
-    vertices[1].m_posf = Vector3f(1, 0, 0);
-    vertices[2].m_posf = Vector3f(0, 1, 0);
-    vertices[3].m_posf = Vector3f(0, 0, 1);
-    std::vector<std::array<size_t, 4>> tets = {{{0, 1, 3, 2}}};
-    std::vector<TetAttributes> tet_attrs(1);
-
-    tetwild.init(vertices.size(), tets);
-    tetwild.create_mesh_attributes(vertices, tet_attrs);
-
-    tetwild.split_all_edges();
-
-	REQUIRE(tetwild.check_mesh_connectivity_validity());
+    // TODO
```

Comment:
Can edge split be tested without tetwild?

Reply:
i think it should, I just moved files around from another PR

Reply:
Let's add this to the TODO list and move on with this PR.